device/samples/DeviceClientTwinSample/Program.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.Azure.Devices.Client.Samples
     using System;
     using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.E2ETests;
```

Comment:
>using Microsoft.Azure.Devices.E2ETests; [](start = 4, length = 39)

We shouldn't require this in our Samples.

Reply:
The original code was reading a value from the `Environment`. Since it's hard to keep track of all environment variables and ensure everything is correctly set, I've unified the test configuration in `Configuration.IoTHub and Configuration.Provisioning`.

I will remove the configuration completely, letting the user to fill-in the correct required properties. 
I suggest we switch to command-line parameters to avoid mistakes where people check-in their SharedAccessKeys, Tokens, Hub addresses, etc.